JSTests/wasm/noJIT/noJIT.js

```diff
@@ -1,2 +0,0 @@
-if (typeof WebAssembly !== "undefined")
```

Comment:
We should make wasm be off by default if you don't have jit support for now (which should let this test pass), and then when we can eventually actually run wasm, we should run this test with --useWebAssembly=0

Reply:
Alrighty, redid the PR accordingly.